30-dictionaries-and-maps.cs

```diff
@@ -14,9 +14,13 @@
 }
 
 for (int i = 0; i < count; i++)
+{
     phoneBookNames.Add(Console.ReadLine().ToLower().TrimEnd());
+}
 
 foreach (var item in phoneBookNames)
+{
     Console.WriteLine(phoneBook.ContainsKey(item) ? item + "=" + phoneBook[item] : "Not found");
```

Comment:
here you two times going throw the dictionary to check the key containing and get its value, I suggest using this 
`Console.WriteLine(phoneBook.TryGetValue(item, out var value) ? item + "=" + value : "Not found");`